man1/aur-chroot.1

```diff
@@ -144,20 +144,10 @@ are checked in order:
 .br
 .BI /usr/share/devtools/makepkg.conf.d/<machine>.conf
 .
-.SS makechrootpkg options
```

Comment:
I'd leave the man page unchanged until `chroot--standalone` becomes the default implementation